packages/awsfirehose/changelog.yml

```
@@ -1,9 +1,14 @@
 # newer versions go on top
+- version: "1.8.1"
+  changes:
+    - description: Update correct link for pull request
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/14691
```

and then change this to the current PR?
```suggestion
      link: https://github.com/elastic/integrations/pull/14845
```